test/usage/dune

```diff
@@ -0,0 +1 @@
+(dirs (:standard \ reason))
```

Comment:
This file is no longer necessary with recent Reason. At least, I didn't need it locally, and it interferes with running the usage tests (which can be worked around, but better remove this file if possible).

Reply:
Nice, I didn't know Reason was compatible with OMP2 already.